pkg/urlfmt/url_test.go

```diff
@@ -38,3 +38,10 @@ func TestGetServerFromURL(t *testing.T) {
 	assert.Equal(t, "localhost", GetServerFromURL("http://localhost"))
 	assert.Equal(t, "localhost:6060", GetServerFromURL("http://localhost:6060/v1"))
 }
+
+func TestTrimHTTPPrefixes(t *testing.T) {
```

Comment:
[nit] I would test also `http://` since this function trims both `http` and `https`

Reply:
should be addressed in [e94d9d2](https://github.com/stackrox/stackrox/pull/11668/commits/e94d9d2ee45970a469fd174cab5fc4b1d4fcdb52)